.github/workflows/nightly_test.yml

```diff
@@ -0,0 +1,205 @@
+name: Nightly Test Corso
+env:
+  IMAGE_NAME: ghcr.io/alcionai/corso
+on:
+  workflow_dispatch:
+
+  pull_request:
```

Comment:
Don't think we should be running on PRs. IIUC, the idea is to only run them on a schedule and not on PRs.